controllers/delete.go

```diff
@@ -20,18 +20,27 @@ import (
 	"context"
 	"fmt"
 
+	ibmv1alpha1 "github.com/IBM/ibm-storage-odf-operator/api/v1alpha1"
 	"github.com/go-logr/logr"
+	ocsv1 "github.com/openshift/ocs-operator/api/v1"
+	odfv1alpha1 "github.com/red-hat-data-services/odf-operator/api/v1alpha1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-
-	ibmv1alpha1 "github.com/IBM/ibm-storage-odf-operator/api/v1alpha1"
-	ocsv1 "github.com/openshift/ocs-operator/api/v1"
-	odfv1alpha1 "github.com/red-hat-data-services/odf-operator/api/v1alpha1"
 )
 
 func (r *StorageSystemReconciler) deleteResources(instance *odfv1alpha1.StorageSystem, logger logr.Logger) error {
 
+	allSSDeleted, err := r.checkStorageSystemDeletionStatus()
+	if err != nil {
+		// Log the error but not fail the operator
+		logger.Error(err, "Failed to List", "Kind", instance.Spec.Kind)
+	}
+
+	if allSSDeleted {
+		r.deleteQuickStarts(logger)
+	}
+
```

Comment:
this should be under deleteQuickStarts func itself.

Reply:
done